app/views/organisations/_show_organisation.html.erb

```diff
@@ -1,9 +1,33 @@
-<div class="organisation-logo">
-  <%= organisation.logo["formatted_title"] %>
-</div>
+<%
+  content_for :breadcrumbs do; end
+%>
+
+<% if @header.is_no_10? %>
+  <h1 class="no10-banner"><%= @header.org.title %></h1>
+
+<% else %>
+  <h1 class="visually-hidden"><%= @header.org.title %></h1>
+  <div class="grid-row">
+    <div class="<%= @header.logo_wrapper_class %>">
+      <%= render "govuk_publishing_components/components/organisation_logo", {
+        organisation: @header.organisation_logo
+      } %>
+    </div>
+
+    <div class="<%= @header.link_wrapper_class %>">
+      <%= render "govuk_publishing_components/components/translation-nav", @header.translation_links %>
```

Comment:
On mobile, the no-margin-top means that there's not enough spacing between this and the logo:

<img width="285" alt="screen shot 2018-06-08 at 16 33 01" src="https://user-images.githubusercontent.com/29889908/41167149-0170c336-6b3a-11e8-8e7f-74e3f158ebad.png">

Reply:
Same with links and organisation logo: 

<img width="192" alt="screen shot 2018-06-08 at 16 37 04" src="https://user-images.githubusercontent.com/29889908/41167215-34db0b8c-6b3a-11e8-96bf-c1710d2009c3.png">

Reply:
Good spot. Annoyingly I'm going to have to put in some glue code for many of the page sections since the margin bottom of the components is inconsistent - the logo has no margin at all. Hopefully I can just do a generic class and extend it, will add.